src/components/label/label.js

```diff
@@ -1,18 +1,23 @@
 import { Link } from "@reach/router";
 import React from "react";
+import "./label.css";
 
-const label = ({ title, label, linkTo = "#" }) => (
+const label = ({ title, Label, linkTo = "#" }) => (
   <div className="label">
     <div className="status">
       <Link to={linkTo}>
         <h1>{title}</h1>
-        {label}
+        {Label}
       </Link>
-      <button>+</button>
+      <div className="challenge_status">+</div>
     </div>
-    <div className="first_button">
-      <button>Learning</button>
-      <button>Ny resolutions</button>
+    <div className="challenge_tags">
```

Comment:
I don't think we need these forms. This component should be only _one_ label, not a list of labels.